### PR TITLE
Added install_requires, removed __version__ import.

### PR DESCRIPTION
**Pull request by gusmaogabriels**
Installing via pip.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,20 @@
 
 ## INSTALLATION
 
+### pip/setup.py
+
+CatMap can be installed directly via pip: 
+
+    pip install --upgrade https://github.com/medford-research/catmap/zipball/master
```
Comment (mhangaard) [inline README.md]: Great that pip install instructions are added. It should really point to the upstream, though.

Comment (gusmaogabriels) [reply README.md]: Well pointed, Martin. I will amend the pull request. Thanks.

**Comment by mhangaard** [comment]
@gusmaogabriels Hi Gabriel, I have given you admin for this repo. You can merge and push.

**Comment by gusmaogabriels** [comment]
> @
Thank you, Martin! I guess we are now ready to merge.

**Pull request merged**